### PR TITLE
Update: Cloudflare.Warp 1.5.461.0

### PR DESCRIPTION
**Pull request by quhxl**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23753)

### CODE REVIEW

**Comment by quhxl** [comment]
Screenshot 2021-08-05 093134

**Review by OfficialEsco — DISMISSED** [review]
@palenshus

**Comment by palenshus** [comment]
> @palenshus Union Palenshus FTE

Thanks for the heads up, it didn't get properly assigned for some reason

**Comment by palenshus** [comment]
When I go to https://one.one.one.one/, the download url points to https://one.one.one.one/Cloudflare_WARP_Release-x64.msi, not https://1.1.1.1/...

I'd like to add a waiver for this one, but the one.one.one.one URL seems to be the correct one to allow

File: manifests/c/Cloudflare/Warp/1.5.461.0/Cloudflare.Warp.installer.yaml [hunk 1]
```diff
@@ -20,10 +20,10 @@ Installers:
   InstallerType: msi
   Scope: machine
   InstallerUrl: https://1.1.1.1/Cloudflare_WARP_Release-x64.msi
```
Comment (palenshus) [inline manifests/c/Cloudflare/Warp/1.5.461.0/Cloudflare.Warp.installer.yaml:22]:
```suggestion
  InstallerUrl: https://one.one.one.one/Cloudflare_WARP_Release-x64.msi
```

**Review by OfficialEsco — APPROVED** [review]
Ohyeah, i remember https://1.1.1.1/ giving issues in the PackageUrl, guessing the InstallerUrl got the same logic now

**Pull request merged**